ts-rs/src/lib.rs

```diff
@@ -169,7 +169,7 @@ use crate::typelist::TypeList;
 
 pub use ts_rs_macros::TS;
 
-pub use crate::export::ExportError;
+pub use crate::export::{ExportError, __private};
```

Comment:
One question. Should a `mod` called `__private` be in a `pub use`? This will allow anyone to do `use ts_rs::__private;`

Reply:
It has to be unfortunately. The macro generated code uses the code in __private. It's annotated with #[doc(hidden)] and the module naming should make it clear that the api is internal.

Reply:
Serde uses a similar pattern: https://github.com/serde-rs/serde/blob/9e680620b595724a6abfb8e785155d02482f911a/serde/src/lib.rs#L311

Reply:
I see, don't worry about it then, I just thought it was odd due to the name, but users should be sensible enough to know that something called `__private` probably shouldn't be used.